index.html

```diff
@@ -10,10 +10,40 @@
 </head>
 <body>
     <header>
-
+        <nav class="navbar">
+            <a href="/" class="welcome">Welcome</a>
+            <img width="30" height="30" src="./assets/menuIcon.svg" alt="menu">
+        </nav>
     </header>
     <main>
-        
+        <section id="headline">
+            <article class="titleContainer">
+                <h1>Hey there. i'm Miguelangel</h1>
+                <h3>I'm a software <br> developer </h3>
+            </article>
+            <article class="headlineDescription">
+                <p> 
+                    If you can dream it, I can code it, I am a front end developer who specializes in mobile and web apps, capable of developing your project or idea, if you require assistance, do not hesitate to contact me
+                </p>
+            </article>
+            <div class="socialLinks">
+                <a href="https://twitter.com/miguelescorched">
+                    <img src="./assets/icons/twitter.svg" alt="twitter">
+                </a>
+                <a href="https://www.linkedin.com/in/miguelangel-escorche-delgado-9a2956163/">
+                    <img src="./assets/icons/linkedin.svg" alt="twitter">
+                </a>
+                <a href="https://miguelangelescorche.medium.com">
+                    <img src="./assets/icons/medium.svg" alt="twitter">
+                </a>
+                <a href="https://github.com/miguelesco">
+                    <img src="./assets/icons/github.svg" alt="twitter">
+                </a>
+                <a href="">
+                    <img src="./assets/icons/angellist.svg" alt="twitter">
+                </a>
+            </div>
```

Comment:
- [ ] OPTIONAL: The social-icons section would have been better wrapped with ul and each a-tag wrapped with li-tag.